doc/harmony-ap_docker_installation_guide.md

```diff
@@ -156,6 +159,7 @@ See the Static Discovery Configuration Guide \[[UG-SDCG](static_discovery_config
 | ADMIN_USER\*                   | harmony           | Initial admin user for admin UI                                                              |
 | ADMIN_PASSWORD\*               | *random*          | Initial admin user password                                                                  |
 | HARMONY_PARAM_FILE             | *n/a*             | Path to (mapped) parameter file                                                              |
+| EXTERNAL_LB                    | false             | Offload TLS termination to an external load balancer. By enabling this option Access Point will listen on port 8080 (HTTP) instead of 8443 (HTTPS) |
 | DEPLOYMENT_CLUSTERED           | false             | Enables clustering support                                                                   |
```

Comment:
It would be good to add the below sentence to the description of the `DEPLOYMENT_CLUSTERED` variable too. It would be good to add it to the Docker installation guide and the clustering user guide.

> By enabling this option Access Point will listen on port 8080 (HTTP) instead of 8443 (HTTPS)